actionlib/package.xml

```diff
@@ -20,6 +22,8 @@
   <author>Mikael Arguedas</author>
 
   <buildtool_depend version_gte="0.5.78">catkin</buildtool_depend>
+  <buildtool_depend condition="$ROS_PYTHON_VERSION == 2">python-setuptools</buildtool_depend>
```

Comment:
Since this is targeting a `noetic-devel` branch, there's no need to include a `<buildtool_depend>` for either `ROS_PYTHON_VERSION` `2` or `3`.  As of ros/catkin#1048 catkin exports the dependency on `python3-setuptools` for the convenience of all downstream python packages.

Reply:
Good to know that. I reverted the `package.xml` changes.